### PR TITLE
Cleanup docs css

### PR DESCRIPTION
**Pull request by maazadeeb**
Related to #3041.

Using the code coverage feature from Chrome devtools, I've figured out the CSS which is never used in the new docs site. Along the way, I also realized that some CSS is outdated due to some DOM structure changes. I've kept note of the CSS which is not used based on DOM structure change and will do that next.

Let me know if you notice something deleted which might be required.

Also, I fixed a small CSS problem with the twitter followers number

TODO:

 - [x] Figure out the navbar CSS related to branding and hover
 - [ ] Things related to `bs-example` class are kind of broken because of change in DOM structure (Because of using react-live I believe). Fix it
 - [x] The RB icon has a bunch of CSS of the actual Bootstrap icon CSS, and it's overridden almost completely. Get rid of the extra styles
 - [x] The prop tables seem to broken on mobile, with extra horizontal scroll coming up for the whole page. This is not there in master. Need to investigate
 - [ ] Fix horizontal scrolling on mobile for
    - [ ] Buttons
    - [ ] Images
    - [ ] Navbar
    - [ ] Overlays
    - [ ] Pagination

Changes that require feedback:
 - Have fallen back to Bootstrap defaults for the colors of the nav links. No additional changes done there as the defaults look pretty good actually. The current pinkish bottom border might require a bit more hacking, if it's actually needed.

### CODE REVIEW

**Comment by maaz93** [comment]
I just noticed that I've deleted some stuff in the navbar which was actually a change in DOM structure. Will introduce it back

**Comment by maaz93** [comment]
As I'm looking to modify the examples CSS, I have this thought: Should there be any external CSS applied to the displayed examples which is not immediately obvious to a consumer, when they look the JSX and the rendered output?

As it stands currently, there is some extra CSS applied to the rendered examples which the consumer of the library will never know unless they see the source. Shouldn't we make it explicit somehow that "Here's the rendered button from react-bootstrap. But we've applied some extra CSS so that all the rendered buttons don't look cluttered on the screen"?

Thoughts?

**Comment by maaz93** [comment]
Hmm. That makes sense.

**Comment by maaz93** [comment]
Apologies for the lack of updates here. I haven't been able to gather the time to finish it off. I can complete the second TODO soon, I think I only have one or two more components.

Can we create a separate issue for the horizontal scrolling issues that I've mentioned in the TODO list?

File: www/src/components/PageFooter.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ class PageFooter extends React.Component {
               <li>
                 <iframe
                   src="https://platform.twitter.com/widgets/follow_button.html?screen_name=react_bootstrap&show_screen_name=true"
-                  width={230}
+                  width={235}
```
Comment (taion) [inline www/src/components/PageFooter.js]: ooo

File: www/src/pages/components/modal.js [hunk 2]
```diff
@@ -24,7 +24,10 @@ export default withLayout(function ModalSection({ data }) {
         Static Markup
       </Heading>
       <p>A modal dialog component</p>
-      <ReactPlayground codeText={ModalStatic} />
+      <ReactPlayground
+        exampleClassName="bs-example-modals-static"
```
Comment (taion) [inline www/src/pages/components/modal.js]: i wonder if we should actually just have the modal dialog as a component... we had people confused by this before

Comment (jquense) [reply www/src/pages/components/modal.js]: Funny thing, that is what we are doing in the example now. This bonus class isn't doing anything, just noticed today

**Comment by maaz93** [comment]
I was having problems merging yesterday. Some error about socket io client or something like that. @jquense were you able to get the docs running locally after the merge?

**Pull request closed without merging**